app/sagas/deepLinking.js

```diff
@@ -180,7 +180,7 @@ const handleOpen = function* handleOpen({ params }) {
 			yield fallbackNavigation();
 			return;
 		}
-		yield put(appStart({ root: ROOT_NEW_SERVER }));
+		yield put(appStart({ root: ROOT_OUTSIDE }));
```

Comment:
I don't know if we should change this, since we're keeping the same root. Can you test?